tsl/test/expected/dist_hypertable-14.out

```diff
@@ -5140,6 +5140,29 @@ SELECT count(*) FROM disttable;
 -- ignore this oid mismatch for non-builtin types.
 SET timescaledb.enable_connection_binary_data=true;
 \set ON_ERROR_STOP 0
+SET timescaledb.remote_data_fetcher = 'rowbyrow';
+SELECT * FROM disttable ORDER BY 1;
```

Comment:
Nit: would be nice if we could compare the output of rowbyrow and cursor programmatically instead of having to do it visually.

Reply:
They shouldn't necessarily be the same. In general, I don't care about output in these tests since it can be influenced by some minor changes in libpq between releases and so on. I'd disable the error output altogether if I could. The main purpose is to get the error handling code some coverage -- check that it doesn't segfault and doesn't break the state of the session somehow.

Reply:
Oops, I was thinking about a different test. For this one, its purpose is to fail in PG <= 13 to test the error handling code. So the comparison of results in 14 is not intended.